pkgs/development/python-modules/pastedeploy/default.nix

```diff
@@ -4,27 +4,30 @@
   fetchFromGitHub,
   pytestCheckHook,
   pythonOlder,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "pastedeploy";
-  version = "3.0.1";
-  format = "setuptools";
+  version = "3.1";
+  pyproject = true;
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "Pylons";
-    repo = pname;
+    repo = "pastedeploy";
     rev = "refs/tags/${version}";
     hash = "sha256-8MNeOcYPEYAfghZN/K/1v/tAAdgz/fCvuVnBoru+81Q=";
```

Comment:
@anthonyroussel you didn't update the hash for the new version (which also means you built and tested against the old version's source, which happened to be in your local store as a fixed-output derivation).

Reply:
Thank you for the review!

Oops sorry, indeed, looks like I forgot to update the hash for this package.

I created this PR to fix the hash: https://github.com/NixOS/nixpkgs/pull/368915